test/Graphics/Drawable.test.cpp

```diff
@@ -7,3 +10,40 @@ static_assert(!std::is_copy_constructible_v<sf::Drawable>);
 static_assert(std::is_copy_assignable_v<sf::Drawable>);
 static_assert(!std::is_nothrow_move_constructible_v<sf::Drawable>);
 static_assert(std::is_nothrow_move_assignable_v<sf::Drawable>);
+
+class DrawableTest : public sf::Drawable
+{
+public:
+    void draw(sf::RenderTarget&, const sf::RenderStates&) const override
+    {
+        m_called += 1;
+    }
+    int callCount() const
+    {
+        return m_called;
+    }
+
+protected:
+    mutable int m_called{};
+};
+
+TEST_CASE("[Graphics] sf::Drawable")
+{
+    SUBCASE("Construction")
+    {
+        const DrawableTest drawableTest;
+        REQUIRE(drawableTest.callCount() == 0);
```

Comment:
```suggestion
        CHECK(drawableTest.callCount() == 0);
```
Prefer using `CHECK` over `REQUIRE` since a failed `CHECK` will let the rest of the tests keep running. Only use `REQUIRE` if you're trying to guard against UB in the tests should a given assertion fail like ensuring the size of a container is greater than 0 before you try to access an element.